plugins/source/hackernews/client/services/backend.go

```diff
@@ -0,0 +1,10 @@
+package services
+
+import "context"
+
+//go:generate mockgen -package=mocks -destination=../mocks/backend.go -source=backend.go BackendClient
+type Backend interface {
+	Set(ctx context.Context, table, key, value string) error
```

Comment:
You should embed the interface (`backend.Backend`) from the plugin SDK here

Reply:
Ah yeah, good idea